nvi-rest/src/main/java/no/sikt/nva/nvi/rest/fetch/FetchNviCandidateHandler.java

```diff
@@ -35,7 +35,7 @@ protected CandidateResponse processInput(Void input, RequestInfo requestInfo, Co
                    .map(UUID::fromString)
                    .map(service::findById)
                    .map(Optional::orElseThrow)
-                   .map(CandidateResponse::fromCandidate)
+                   .map(candidate -> CandidateResponse.fromCandidate(candidate, service))
```

Comment:
I think it is not the best implementation, but otherwise i have to fetch candidate and period on the top of the method and pass it here.